Source/Core/DolphinQt2/NetPlay/PadMappingDialog.cpp

```diff
@@ -56,7 +56,7 @@ int PadMappingDialog::exec()
   players.append(tr("None"));
 
   for (const auto& player : m_players)
-    players.append(QString::fromStdString(player->name));
+    players.append(tr("%1 (%2)").arg(QString::fromStdString(player->name)).arg(player->pid));
```

Comment:
I don't think this string needs to be translated. Maybe use ``QStringLiteral`` instead?

Reply:
Hmm, right, it's not like there's anything that can be translated in a meaningful way. Fixed